vm/src/rv32/parse.rs

```diff
@@ -241,6 +241,15 @@ fn inst_size(b0: u8, b1: u8) -> u32 {
     }
 }
 
+/// translate RV32i instructions to RV32Nexus instructions
+fn translate_nexus(word: u32) -> u32 {
+    match word {
+        // ecall     // ebreak
+        0x00000073 | 0x00100073 => word | (0b101 << 8), // set rd = 10
```

Comment:
nit: The relationship between 10 and the instruction encoding of `rd` in bits 7-11 would be clearer like this, at least to me.

```suggestion
        0x00000073 | 0x00100073 => word | (0b1010 << 7), // set rd = 10
```